README.md

```diff
@@ -176,12 +176,17 @@ Instead we need to modify layout A's template, and mark the ``description`` anno
 Running Portia
 --------------
 
-Checkout the repository:
+### Dependencies ###
+You might need to install the following packages
 
-    git clone https://github.com/scrapinghub/portia
+    sudo apt-get install libxml2-dev libxslt-dev python-dev zlib1g-dev libffi-dev libssl-dev
```

Comment:
I think build-essential should be here too.

Are you sure about the zlib dependency? Is so do you know which library requires it?